README.markdown

```diff
@@ -4883,6 +4883,11 @@ ngx.req.set_header
 
 Set the current request's request header named `header_name` to value `header_value`, overriding any existing ones.
 
+The input Lua string header_name and header_value should already be well-formed with the URI encoding.
```

Comment:
Missing backticks around `header_name` and `header_value`. See the previous paragraph for examples.